src/components/Header.jsx

```diff
@@ -73,14 +73,16 @@ const AuthZoomButton = ({ t }) => {
             aria-controls="menu-appbar"
             aria-haspopup="true"
             onClick={(e) => setAnchorEl(e.currentTarget)}
-            color="inherit">
+            color="inherit"
+            sx={{ height: '100%', width: 64 }}>
```

Comment:
`height` added to prevent slight vertical shift in links when logging in / logging out
`width` added to prevent horizontal shift in links when logging in / logging out